Packs/AzureKeyVault/Integrations/AzureKeyVault/AzureKeyVault.py

```diff
@@ -564,70 +624,91 @@ def create_or_update_key_vault_command(client: KeyVaultClient, args: Dict[str, A
     ignore_missing_vnet_service_endpoint = argToBoolean(
         args.get('ignore_missing_vnet_service_endpoint', True))
     ip_rules = argToList(args.get('ip_rules'))
-
-    response = client.create_or_update_key_vault_request(vault_name, object_id, location, sku_name, keys_permissions,
-                                                         secrets_permissions, certificates_permissions,
-                                                         storage_accounts_permissions, enabled_for_deployment,
-                                                         enabled_for_disk_encryption, enabled_for_template_deployment,
-                                                         default_action, bypass, vnet_subnet_id,
-                                                         ignore_missing_vnet_service_endpoint, ip_rules)
+    subscription_id = args.get('subscription_id', params.get('subscription_id'))
```

Comment:
Please add a comment explaining the idea here - that we use the arg and only when it is not specified we use the integration param.

Also is it possible to get here without any of them? 

Relevant to all commands.

Reply:
1.Done.
2.Yes, its possible, but the test module will fail.